packages/protocol/contracts/AccountRegistry/epochs/20200207/Behaviour20200207.sol

```diff
@@ -0,0 +1,92 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+import "../20200106/Behaviour20200106.sol";
+
+/**
+ * @title Behaviour20200207 implementation
+ * @author AZTEC
+ * Note the behaviour contract version naming convention is based on the date on which the contract
+ * was created, in the format: YYYYMMDD
+ * 
+ * Copyright 2020 Spilsbury Holdings Ltd 
+ *
+ * Licensed under the GNU Lesser General Public Licence, Version 3.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ **/
+contract Behaviour20200207 is Behaviour20200106 {
+    /**
+    * @dev epoch number, used for version control in upgradeability. The naming convention is based on the
+    * date on which the contract was created, in the format: YYYYMMDD
+    */
+    uint256 public epoch = 20200207;
+
+    /**
+    * @dev The updated trustedGSNSigner address, to be used in acceptRelayedCall()
+    */
+    address public GSNSigner;
+
+    /**
+    * @dev Set the GSN signer address for this behaviour contract
+    */
+    function setGSNSigner() public {
```

Comment:
Is it worth us just defining this in `acceptRelayCall` just so we reduce the number of things we have to do in a migration? The variable isn't used elsewhere.

Reply:
As discussed, leaving as is